### DISCUSSION TITLE
167

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#120] [General]
# 潮流周刊第167期 - 碧空雪涛

封面图来源于去年冬天从新疆上空飞长沙的飞机上拍摄乌鲁木齐的雪景，当时早上 9 点多的样子，看到远处的远处的阳光，远处的蓝天，远处的云，远处的雪，远处的山，真的是太美了，用碧空雪涛形容很合适。

https://weekly.tw93.fun/posts/167-%E7%A2%A7%E7%A9%BA%E9%9B%AA%E6%B6%9B/

**Comment by michaelliuyang** [comment]
虚实之间，非常赞。我也非常喜欢，安利了周围很多朋友，每期必听，但是播客订阅量和内容输出质量严重不符。

**Reply by tw93** [reply]
是的 可能是新号的缘故

**Comment by caix-github** [comment]
建议点击链接跳转新页签，每次点击后总要回退回来

**Reply by tw93** [reply]
现在应该就是的，你说的是哪一个地址

**Reply by caix-github** [reply]
好像是看往期的周刊有些会有，不过现在看最新的都有跳转了

**Comment by HarveySuen0803** [comment]
咖啡，冰淇淋也可以咸鱼，很便宜，哈哈哈

**Reply by tw93** [reply]
哈哈 下次买东西前先看

**Reply by QuakeWang** [reply]
麦麦也可以闲鱼 哈哈

**Comment by CNife** [comment]
我也经常买咸鱼低价电影票，还有话费慢充、会员福利这些东西。

**Reply by xh-0** [reply]
赞同，去年买的b站大会员70一年，太实惠了

**Reply by zcyc** [reply]
大家注意，话费慢充涉嫌洗钱哦

**Comment by Levix** [comment]
https://weekly.tw93.fun/rss.xml
rss 源挂了。

**Reply by tw93** [reply]
很感谢反馈，再试试看，应该好了

**Reply by Levix** [reply]
可以用了。

**Comment by janebooom** [comment]
times.is用的是系统的时间吗？还是远程的？
